wemake_python_styleguide/logic/nodes.py

```diff
@@ -29,3 +32,36 @@ def get_parent(node: ast.AST) -> Optional[ast.AST]:
 def get_context(node: ast.AST) -> Optional[ContextNodes]:
     """Returns the context or ``None`` if node has no context."""
     return getattr(node, 'wps_context', None)
+
+
+def does_loop_contain_node(  # TODO: move, reuse in annotations.py
```

Comment:
Do we actually use this method from outside? If not, it should be protected. `_does_loop_contain_node`

Reply:
Done!